File: C/Iniciante/1185/1185.c
#include <stdio.h>
int main()
{
    double soma = 0.0, M[12][12];
    char O;
    int coluna, linha, contagem = 10;
    scanf("%s", &O);
    for (linha = 0; linha <= 11; linha++)
    {
        for (coluna = 0; coluna <= 11; coluna++)
            scanf("%lf", &M[linha][coluna]);
    }
    for (linha = 0; linha <= 10; linha++)
    {
        for (coluna = 0; coluna <= contagem; coluna++)
            soma += M[linha][coluna];
        contagem--;
    }
    if (O == 'S')
        printf("%.1lf\n", soma);
    else
    {
        printf("%.1lf\n", soma / 66.0);
    }
    return 0;
}

File: Python/Iniciante/1185.py
# Problema 1185 - Beecrowd - Iniciante - Nível 2

# Iniciamos a matriz, que será uma lista de listas, a soma 
# se iniciará com 0 e recebmos a operação a ser realizade
matriz = []
soma = 0
operacao = input()

# Preenchemos a cada linha e depois, utilizamos 'append'
# para inserir a linha na matriz
for _ in range(12):
    linha = []
    for _ in range(12):
        linha.append(float(input()))
    matriz.append(linha)

# Percorremos a matriz, e quando a soma da linha com a coluna
# for menor ou igual à 10, representa que o elemento está na 
# diagonal superior e então, é adicionado a variável SOMA
for i in range(12):
    for j in range(12):
        if (i + j) <= 10:
            soma += matriz[i][j]

# De acordo com a operação, o resultado é exibido com apenas uma
# casa decimal
if operacao == 'S':
    print(f'{soma:.1f}')
else:
    print(f'{soma/66:.1f}')